src/app.d.ts

```diff
@@ -1,3 +1,5 @@
+/// <reference types="@sveltejs/kit" />
```

Comment:
This was needed to be able to `import { browser } from '$app/environment'`

Reply:
I am a bit cautious since this may allow us to use `browser` in the components as well. We don't want that and should be quite mindful about it.

Reply:
Okay, I created an `index.d.ts` in `src/lib/utils` containing this line, so it only applies there.